bundles/paikkatietoikkuna/coordinatetransformation/components/SourceSelect.js

```diff
@@ -0,0 +1,70 @@
+Oskari.clazz.define('Oskari.coordinatetransformation.component.SourceSelect',
+    function (loc) {
+        var me = this;
+        me.loc = loc;
+        this.element = null;
+        me._template = {
+            sourceWrapper: jQuery('<div class="sourceWrapper"></div>'),
+            source: _.template(
+                '<div class="coordinate-datasource"> </br> ' +
+                '<h4><%= title %></h4>'+
+                '<form>'+
+                    '<input type="radio" id="clipboard" name="load" value="2"><label for="clipboard"><span></span> ${clipboard} </label>'+
+                    '<input type="radio" id="file" name="load" value="1"><label for="file"> <span></span> ${file} </label>'+
+                    '<input type="radio" id="mapselection" name="load" value="3"><label for="mapselection"> <span></span> ${map} </label>'+
+                '</form>'+
+                '</div>'
+            ),
+            info: _.template(
+                '<div class="datasource-info">' +
+                    '<div class="coordinateconversion-clipboardinfo" style=display:none;">'+
+                        '<div class="clipboardinfo"> <i><%= clipboardupload %><i> </div>'+
+                    '</div>' +
+                    '<div class="coordinateconversion-mapinfo" style=display:none;">'+
+                        '<div class="mapinfo">'+
+                            '<i>${mapinfo}<i> </br>'+
+                            '<input type="button" class="selectFromMap" name="load" value="${map}">' +
+                        '</div>' +
+                    '</div>' +
+                '</div>'
+            )
+        }
+        if (this.element === null) {
```

Comment:
This is always true so you can remove it. Or add if (this.element !== null) { return; } at the beginning of createUi()